internal/controller/promotions/promotions.go

```diff
@@ -414,12 +415,8 @@ func (r *reconciler) Reconcile(
 		return ctrl.Result{}, err
 	}
 
-	// If the promotion is still running, we'll need to periodically check on
-	// it.
```

Comment:
We can lose the TODO below, but I think this bit of the comment is worth keeping.